examples/contour/01-contour-config.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   namespace: projectcontour
 data:
   contour.yaml: |
+    # determine which XDS Server implementation to utilize in Contour.
+    # Options are: "contour" or "gocontrolplane"
```

Comment:
Nit: Just a note here that "gocontrolplane" is TBD?

Reply:
I'll take out since it's not implemented yet, but the next one will probably implement. We'll need names, if that's `contour` & `gcp` or `gocontrolplane` or something like `builtin` & `upstream`. Or `stable` & `experimental`? I dunno, but that can be for the next one. =)

Reply:
`gcp` is out because `Google Compute Platform` or whatever it actually stands for. ;)